go.mod

```diff
@@ -7,4 +7,7 @@ require (
 	k8s.io/klog/v2 v2.90.1
 )
 
-require github.com/go-logr/logr v1.2.3 // indirect
+require (
+	github.com/go-logr/logr v1.2.3 // indirect
+	k8s.io/klog v1.0.0 // indirect
```

Comment:
We use k8s.io/klog/v2 in other search repos. Maybe good to keep it consistent for CVEs and other stuff.